apps/webapp/components/shared/media-sections.tsx

```diff
@@ -28,8 +28,8 @@ export function MediaSections({ sections, lang }: MediaSectionsProps) {
               </div>
 
               {/* <ul className="grid-cols-auto-dense grid w-full grid-cols-[repeat(auto-fill,minmax(250px,1fr))] flex-col gap-5 py-5 sm:flex-wrap"> */}
-              <ul className="grid w-full grid-cols-[repeat(auto-fill,minmax(250px,1fr))] flex-col gap-5 py-5 sm:flex-wrap">
-                {section?.videos?.map(video => (
+              <ul className="grid w-full grid-cols-[repeat(auto-fill,minmax(288px,1fr))] flex-col gap-8 py-5 sm:flex-wrap">
+                {section?.videos?.slice(0, 4)?.map(video => (
```

Comment:
Essentially this is very good, it is controlling the maximum content to show on these media sections however, this should be dynamic by screensize:

- To show up to 2 cards on mobile (max-width: 420px)
- To show up to 4 cards on tablet/small-screens (max-width: 1366px) [md]
- To show up to 5 cards on big screens (min-width: 1366px) [xl]

You can hide cards by using CSS, no JS is required.*

> ***PRO TIP:** display: none; makes an element unaccessible (non a11y) which is very good due the navigation won't be affected. If you map every element on the array to add the `hidden` class can do the job.